modin/core/execution/dask/common/utils.py

```diff
@@ -32,6 +33,14 @@ def initialize_dask():
 
     try:
         client = default_client()
+
+        def _disable_warnings():
+            import warnings
+
+            warnings.simplefilter("ignore", category=FutureWarning)
+
+        client.run(_disable_warnings)
```

Comment:
Why do we need to disable the warnings here, whereas we use the context manager below to enable the warnings?

Reply:
In this section we connect to the already created Dask cluster. Using an environment variable will not help here, since the processes are already running.

The context manager below does not enable warnings, but rather disables a certain category of warnings using an environment variable.